### PR TITLE
Accessibility via aria hidden on cells.

### PR DESCRIPTION
**Pull request by 2474**
Upon instantiation each cell should now have `aria-hidden` set to true. When Flickity is activated and a cell is selected via `setSelectedCell` the `aria-hidden` attribute will be toggled to false. Upon cell change the `aria-hidden` attribute will be toggled to true.

I've also moved the n* classes in `sandbox/basic.html` into the class attribute as I believe that is where they should have been.

Finally I swapped out `gulp-minify-css` with `gulp-clean-css` because it's deprecated.

Hope this helps, this is really the first time I've contributed to anything so if I'm way off feel free to let me know. Would love to learn more and help out where I can.

### CODE REVIEW

**Comment by desandro** [comment]
Thank you for this contribution! Great job on submitting your first PR!

I'm not quite ready to add this feature. See https://github.com/metafizzy/flickity/issues/66#issuecomment-204183302 about the larger discussion for this feature.

I'll follow up with additional comments about the code.

File: gulpfile.js [hunk 1]
```diff
@@ -146,14 +146,14 @@ gulp.task( 'uglify', [ 'requirejs' ], function() {
...
-var minifyCSS = require('gulp-minify-css');
+var cleanCSS = require('gulp-clean-css');
 
 gulp.task( 'css', function() {
   gulp.src('css/flickity.css')
     // copy to dist
     .pipe( gulp.dest('dist') )
     // minify
-    .pipe( minifyCSS({ advanced: false }) )
+    .pipe( cleanCSS({ advanced: false }) )
```
Comment (desandro) [inline gulpfile.js]: Thanks for this! Didn't know about the deprecation. As this is a separate feature, I would have submitted this bit of code in a separate PR, so it doesn't get hung up in this one.

File: js/cell.js [hunk 2]
```diff
@@ -45,6 +45,10 @@ Cell.prototype.create = function() {
   }
   this.x = 0;
   this.shift = 0;
+  // default each cell to aria hidden
+  // determination of selected cell
+  // will change this state
+  this.element.setAttribute('aria-hidden', true);
```
Comment (desandro) [inline js/cell.js]: Why not use `setAriaHidden`?

File: js/cell.js [hunk 3]
```diff
@@ -64,6 +68,10 @@ Cell.prototype.setPosition = function( x ) {
   this.renderPosition( x );
 };
 
+Cell.prototype.setAriaHidden = function ( val ) {
+  this.element.setAttribute('aria-hidden', val);
+};
```
Comment (desandro) [inline js/cell.js]: Adhering to project's code style is a nice way to partner into the project. I use spaces in parens, no space after `function`

``` js
Cell.prototype.setAriaHidden = function( val ) {
  this.element.setAttribute( 'aria-hidden', val );
};
```

File: js/cell.js (hunk 2 above)
Comment (2474) [inline js/cell.js]: Good point, I guess I got a bit overwhelmed and wasn't sure I could use that there. That definitely would be the way to go though.

File: js/cell.js (hunk 3 above)
Comment (2474) [inline js/cell.js]: Oh ok, I tried to adhere to the stuff inside the parens but totally missed the spacing between the functions, thanks for pointing that out!

File: gulpfile.js (hunk 1 above)
Comment (2474) [inline gulpfile.js]: No problem, in the past I've used `gulp-cssnano` but it seems like `gulp-clean-css` is far more used, so I assumed it was probably a better option. True, I probably should have made a pull request, rookie mistakes. Ha ha.
